tests/drivers/rtc/rtc_api/boards/b_u585i_iot02a.conf

```diff
@@ -0,0 +1 @@
+CONFIG_RTC_CALIBRATION=y
```

Comment:
Or we could have a testcase item with EXTRA_CONFIGS for that

Reply:
Not all boards support RTC calibration. Also we would need to maintain the list to insure that the calibration test is not run in multiple test cases.